src/frontend/get-github-configuration.ts

```diff
@@ -176,7 +198,7 @@ export default async (req: Request, res: Response, next: NextFunction): Promise<
 			jiraHost,
 			client,
 			installationsWithAdmin,
-			req.log
+			log
```

Comment:
Made it more consistent which what we have in the function (log is the chold of req.log but with Jira Host parameter)